Mathlib/CategoryTheory/Sites/Sheaf.lean

```diff
@@ -386,6 +386,9 @@ instance : (sheafToPresheaf J A).Full :=
 instance : (sheafToPresheaf J A).Faithful :=
   (fullyFaithfulSheafToPresheaf J A).faithful
 
+instance : (sheafToPresheaf J A).ReflectsIsomorphisms :=
+  (fullyFaithfulSheafToPresheaf J A).reflectsIsomorphisms
```

Comment:
Do we not have the instance `[Full F] [Faithful F] : ReflectsIsomorphisms F`? We should add one.

Reply:
At some point working on this PR, this instance was not found automatically, but now, it seems it works :-)